codec/reflectcodec/type_codec.go

```diff
@@ -495,8 +521,13 @@ func (c *genericCodec) Unmarshal(bytes []byte, dest interface{}) error {
 }
 
 // Unmarshal from p.Bytes into [value]. [value] must be addressable.
+//
+// The `nullable` property affects how pointers and interfaces are unmarshalled,
```

Comment:
We typically use `[]` and not `` 

```suggestion
// The [nullable] property affects how pointers and interfaces are unmarshalled,
```